subprojects/docs/src/docs/userguide/migrating_from_maven.adoc

```diff
@@ -41,11 +41,17 @@ This may make migrating between the two seem intimidating, but migrations can be
 
 Here we lay out a series of steps for you to follow that will help facilitate the migration of any Maven build to Gradle:
 
- 1. Keep the old Maven build and new Gradle build side by side
-+
+TIP: Keep the old Maven build and new Gradle build side by side.
 You know the Maven build works, so you should keep it until you are confident that the Gradle build produces all the same artifacts and otherwise does what you need.
 This also means that users can try the Gradle build without getting a new copy of the source tree.
- 2. Develop a mechanism to verify that the two builds produce the same artifacts
+
+ . link:https://scans.gradle.com[Create a build scan for the Maven build].
```

Comment:
Surely we'll have a Maven-specific page right?

Reply:
Everything I read said it will be there, but maybe I'll get an anchor to link to? Either way, this PR is blocked until that page is ready.

Reply:
updated.